.ci/jobs/defaults.yml

```diff
@@ -3,7 +3,7 @@
 ##### GLOBAL METADATA
```

Comment:
Review line https://github.com/elastic/e2e-testing/pull/156/files#diff-c35ec66bcd810edcb08ef01abec77275L18 since it might not be required to avoid the evil indexing

Reply:
I'm removing the value, as suggested in Slack